### PR TITLE
test/pybind/test_rados.py: test test_aio_remove

### PR DESCRIPTION
**Pull request by zjcmszh**
Signed-off-by: Zhang Jiao <zhangjiao@cmss.chinamobile.com>

### CODE REVIEW

**Comment by zjcmszh** [comment]
@batrick   Please review it for me. Thanks

File: src/test/pybind/test_rados.py [hunk 1]
```diff
@@ -701,6 +701,24 @@ def cb(_, size, mtime):
...
+	lock = threading.Condition()
+	count = [0]
+	def cb(blah):
+	    with lock:
+	        count[0] += 1
+		lock.notify()
+	    return 0
+	self.ioctx.write('foo', b'wrx')
+	eq(self.ioctx.read('foo'), b'wrx')
+	comp = self.ioctx.aio_remove('foo', cp, cp)
```
Comment (tchaikov) [inline src/test/pybind/test_rados.py]: s/cp/cb/

and i wonder how you test this change?

**Comment by zjcmszh** [comment]
@tchaikov  It has been revised.   
Thanks for your advice.

File: src/test/pybind/test_rados.py [hunk 2]
```diff
@@ -701,6 +701,24 @@ def cb(_, size, mtime):
...
+                count[0] += 1
+                lock.notify()
+            return 0
+        self.ioctx.write('foo', b'wrx')
+        eq(self.ioctx.read('foo'), b'wrx')
+        comp = self.ioctx.aio_remove('foo', cb, cb)
+        comp.wait_for_complete()
+        with lock:
+            while count[0] < 2:
+            lock.wait()
```
Comment (tchaikov) [inline src/test/pybind/test_rados.py]: bad indent. 

@zjcmszh please test the change before pushing it.

**Comment by zjcmszh** [comment]
@tchaikov   very very sorry...
It‘s all right this time.

**Pull request merged**